### PR TITLE
args file

### PR DESCRIPTION
**Pull request by conniely04**
args for database, host, port

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -4,21 +4,26 @@
...
+from args import get_args
 
 app = FastAPI()
+args = get_args()
 
 urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
 http = urllib3.PoolManager(cert_reqs='CERT_NONE', assert_hostname=False)
 
-DB_FILE = 'parking.db'
+DB_FILE = None
```
Comment (pnavab) [inline server.py]: how about setting this to args.dbfile instead and using this variable in all the helper functions

Comment (conniely04) [reply server.py]: fixed

File: args.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        default=8000,
+        help="port for server to be hosted on, defaults to 8000"
+    )
+
+    parser.add_argument(
+        "--reload",
+        type=bool,
+        required=False,
+        help="boolean for the uvicorn run command's 'reload' parameter"
+    )
```
Comment (pnavab) [inline args.py]: we dont actually  need this it was just in the example

File: server.py [hunk 3]
```diff
@@ -44,5 +49,21 @@ async def get_garage_data():
...
+    format="%(asctime)s.%(msecs)03dZ %(levelname)s:%(name)s:%(message)s",
+    datefmt="%Y-%m-%dT%H:%M:%S",
+    level= logging.ERROR - (args.verbose*10),
+)
+
 if __name__ == "__main__":
-    uvicorn.run("server:app", port=8000, reload=True)
\ No newline at end of file
+    args = get_args()
+    uvicorn.run("server:app", host=args.host, port=args.port, reload=args.reload, )
```
Comment (pnavab) [inline server.py]:
```suggestion
    uvicorn.run("server:app", host=args.host, port=args.port, reload=True, )
```

File: server.py [hunk 4]
```diff
@@ -44,5 +49,21 @@ async def get_garage_data():
     return garage_data
 
 
+@app.get("/logging-levels")
+def logging_levels():
+    logging.error("error message")
+    logging.warning("warning message")
+    logging.info("info message")
+    logging.debug("debug message")
```
Comment (pnavab) [inline server.py]: this is also from example we dont need this endpoint

Comment (conniely04) [reply server.py]: ok i deleted both parts

**Review by pnavab — APPROVED** [review]
good

**Pull request merged**